app/templates/layouts/basic.layout.twig

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        {% block head %}
+            <meta charset="UTF-8">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+            <title>{% block title %}{% endblock %}</title>
+
+            <link rel="stylesheet" href="web-assets/css/bootstrap.css"/>
+            <link rel="stylesheet" href="web-assets/css/w3.css"/>
+        {% endblock %}
+    </head>
+    <body class="bg-light">
+        {{ include('components/nav.twig') }}
```

Comment:
INCLUDE !!! Woo-hoo! 
This lets us break down the layout into small files. You can `include` templates in templates of templates!